app/src/main/java/com/nickytoolchick/agraph/data/ChartOptions.kt

```diff
@@ -0,0 +1,16 @@
+package com.nickytoolchick.agraph.data
+
+import java.io.Serializable
+
+class ChartOptions() : Serializable {
```

Comment:
For Kotlin development, it is recommended to use the kotlinx.serialization library instead of the java.io.Serializable interface. kotlinx.serialization has more efficient and flexible serialization of data, provides a flexible serialization configuration through annotations and parameters, allowing you to accurately control which fields should be serialized and how they should be represented in the data.

* Add dependency in build.gradle:
```Kotlin
plugins {
    kotlin("plugin.serialization") version "<version>"
}

dependencies {
    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:<version>")
}

```
* Define the ChartOptions class that will be serialized:
```kotlin
import kotlinx.serialization.Serializable

@Serializable
class ChartOptions(){}
```

Reply:
Changed the java.io.Serializable interface to kotlinx.serialization library.